src/bucket.ts

```diff
@@ -32,6 +32,8 @@ export interface S3BucketConfig extends S3Config {
   bucket: string;
 }
 
+export type DeletedKeys = string[];
```

Comment:
I don't think this type alias is needed.

Reply:
I'd like to keep it nonetheless, it adds semantic to what's being returned beyond "it's a list of strings" and it's consistent with the other methods that all have typed returns, this one is just a lot simpler

Reply:
I guess. I would much rather like to communicate things like this via JS doc comments though. doc.deno.land looks a lot cleaner when it returns `string[]` with a JS Doc saying `Deletes all objects in the bucket recursively. Returns a list of deleted keys.`

Reply:
sounds like a good compromise, I'll get that done as soon as I can